continuationsPlugin/src/main/scala/continuations/Continuation.scala

```diff
@@ -20,17 +20,14 @@ object Continuation:
   type Result = Either[Throwable, Any | Null | State.Suspended.type]
 
   def checkResult(result: Result): Unit =
-    result match {
-      case null => ()
-      case Left(ex) => throw ex
-      case Right(_) => ()
-    }
+    if (result ne null) result.void
 
 end Continuation
 
 inline def BuildContinuation[T](
     ec: ExecutionContext,
-    res: Either[Throwable, T] => Unit): Continuation[T] =
+    rai: Throwable => Unit,
+    res: T => Unit): Continuation[T] =
```

Comment:
Need clearer names here, otherwise I think it is ok.

Reply:
renamed as `onResume` and `onRaise`,